tests/test_get_replica_connections.py

```
@@ -0,0 +1,84 @@
+import time
+import pytest
+import subprocess
```

Ruff (0.8.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)